modules/connected-to-disconnected-config-registry.adoc

```diff
@@ -0,0 +1,182 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/connected-to-disconnected.adoc
+
+[id="connected-to-disconnected-config-registry_{context}"]
+= Configuring the mirror registry
+
+After creating the mirror registry and mirroring the images to the registry, you need to make some modifications to your cluster in order for pods to be able to pull images from the mirror registry. 
+ 
+You must add the mirror registry credentials to the global pull secret, add the mirror registry server certificate to the cluster, and create an `ImageContentSourcePolicy` custom resource that associates the mirror registry with the source registry.
+
+. Add mirror registry credential to cluster global pull-secret:
++
+[source,terminal]
+----
+$ oc set data secret/pull-secret -n openshift-config --from-file=.dockerconfigjson=<pull_secret_location> <1>
+----
+<1> Provide the path to the new pull secret file.
++
+For example:
++
+[source,terminal]
+----
+$ oc set data secret/pull-secret -n openshift-config --from-file=.dockerconfigjson=.dockerconfigjson
+----
+
+. Add the CA signed mirror registry server certificate to the nodes in the cluster:
+
+.. Create a config map that includes the server certificate for the mirror registry
++
+[source,terminal]
+----
+$ oc create configmap <config_map_name> --from-file=<mirror_address_host>..<port>=$path/ca.crt -n openshift-config
+----
++
+For example:
++
+[source,terminal]
+----
+S oc create configmap registry-config --from-file=172.31.249.15..443=/root/certs/ca-chain.cert.pem -n openshift-config
+----
+
+.. Use the config map to update the `image.config.openshift.io/cluster` custom resource (CR). {product-title} applies the changes to this CR to all nodes in the cluster:
++
+[source,terminal]
+----
+$ oc patch image.config.openshift.io/cluster --patch '{"spec":{"additionalTrustedCA":{"name":"<config_map_name>config"}}}' --type=merge
+----
++
+For example:
++
+[source,terminal]
+----
+$ oc patch image.config.openshift.io/cluster --patch '{"spec":{"additionalTrustedCA":{"name":"registry-config"}}}' --type=merge
+----
+
+. Create an ICSP to redirect container pull requests from the online registires to the mirror registry:
+
+.. Create the `ImageContentSourcePolicy` custom resource:
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1alpha1
+kind: ImageContentSourcePolicy
+metadata:
+  name: mirror-ocp
+spec:
+  repositoryDigestMirrors:
+  - mirrors:
+    - 172.31.249.15:443/ocp/release <1>
+    source: quay.io/openshift-release-dev/ocp-release <2>
+  - mirrors:
+    - 172.31.249.15:443/ocp/release
+    source: quay.io/openshift-release-dev/ocp-v4.0-art-dev
+----
+<1> Specifies the name of the mirror image registry and repository.
+<2> Specifies the online registry and repository containing the content that is mirrored.
+
+.. Create the ICSP object:
```

Comment:
I think we should add a note that ICSP changes are applied to all the nodes in the cluster

Reply:
@mburke5678 we should add a note here mentioning ICSP changes roll out to all the nodes in the cluster.

Reply:
@umohnani8 See [Line 92 below](https://github.com/openshift/openshift-docs/pull/36351/files#diff-a90477416fef252041fabb5a89b6aa0dabf0b7e7e2fd3e29ae56da9163972cc4R92).